### ISSUE TITLE
Add content of LICENSE and CONTRIBUTING.md files to documentation

### ISSUE THREAD

**Issue by hendrikebbers** [canoo/dolphin-platform#661]
The LICENSE and CONTRIBUTING.md files are part of the Dolphin Platform repository but currently they are not in the asciidoc based documentation. The current text of this files should be added as additional chapters to the end of the documentation.

**Comment by cristh18** [comment]
I don't understand very well, what I should do in this issue. For this files LICENSE and CONTRIBUTING.md, I should pass to asciidoc format with the same text?

**Comment by hendrikebbers** [comment]
Yes. And it should be part in the generated documentation as separate chapters at the end

**Comment by dhrubo55** [comment]
can i work on this issue
